tests/unit/field/test_field2d.cxx

```diff
@@ -117,6 +117,56 @@ TEST_F(Field2DTest, CreateOnGivenMesh) {
   delete fieldmesh;
 }
 
+TEST_F(Field2DTest, CopyCheckFieldmesh) {
+  int test_nx = 2;
```

Comment:
I assume these values are different from the "global" mesh so we're testing that it's definitely using fieldmesh? Would it be more robust to explicitly define these in terms of some difference from the global mesh (e.g. test_nx = global_nx + 2 or similar)?

Reply:
Yes, that would probably make it more obvious what's happening, thanks